tide_event.install

```diff
@@ -425,3 +425,167 @@ function tide_event_update_8009() {
     }
   }
 }
+
+/**
+ * Create field status.
+ */
+function tide_event_update_8010() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_event') . '/config/install'];
+  _tide_import_single_config('field.storage.node.field_status', $config_location);
```

Comment:
Hi @asarfara 
please see the comment on https://github.com/dpc-sdp/tide_landing_page/pull/99/files#r486703935